lib/std/zig/system.zig

```diff
@@ -127,7 +127,9 @@ pub fn getExternalExecutor(
             if (options.allow_wine) {
                 // x86_64 wine does not support emulating aarch64-windows and
                 // vice versa.
-                if (candidate.cpu.arch != builtin.cpu.arch) {
+                if (candidate.cpu.arch != builtin.cpu.arch and
+                    !(candidate.cpu.arch == .x86 and builtin.cpu.arch == .x86_64))
```

Comment:
```suggestion
                if (candidate.cpu.arch != builtin.cpu.arch and
                    !(candidate.cpu.arch == .thumb and builtin.cpu.arch == .aarch64) and
                    !(candidate.cpu.arch == .x86 and builtin.cpu.arch == .x86_64))
```

I would assume that Wine on aarch64 can also run thumb binaries?